src/main/java/com/google/devtools/build/lib/rules/java/BUILD

```diff
@@ -79,6 +79,19 @@ java_library(
     ],
 )
 
+java_library(
```

Comment:
It's a bad practice to have a source file part of two rules. Please remove `JavaCompileActgionContext.java` from the other rule where it's still used and fix dependencies if needed.